display_list/display_list_canvas_unittests.cc

```diff
@@ -50,29 +50,29 @@ constexpr SkRect RenderBounds =
 
 // The X offsets which will be used for tall vertical diamonds are
 // expressed in terms of the rendering height to obtain the proper angle
-constexpr SkScalar MiterExtremeDiamondOffsetX = RenderHeight * 0.04;
-constexpr SkScalar Miter10DiamondOffsetX = RenderHeight * 0.051;
-constexpr SkScalar Miter4DiamondOffsetX = RenderHeight * 0.14;
+constexpr SkScalar kMiterExtremeDiamondOffsetX = kRenderHeight * 0.04;
+constexpr SkScalar kMiter10DiamondOffsetX = kRenderHeight * 0.051;
+constexpr SkScalar kMiter4DiamondOffsetX = kRenderHeight * 0.14;
 
 // The Y offsets which will be used for long horizontal diamonds are
 // expressed in terms of the rendering width to obtain the proper angle
-constexpr SkScalar MiterExtremeDiamondOffsetY = RenderWidth * 0.04;
-constexpr SkScalar Miter10DiamondOffsetY = RenderWidth * 0.051;
-constexpr SkScalar Miter4DiamondOffsetY = RenderWidth * 0.14;
+constexpr SkScalar kMiterExtremeDiamondOffsetY = kRenderWidth * 0.04;
+constexpr SkScalar kMiter10DiamondOffsetY = kRenderWidth * 0.051;
+constexpr SkScalar kMiter4DiamondOffsetY = kRenderWidth * 0.14;
 
 // Render 3 vertical and horizontal diamonds each
 // designed to break at the tested miter limits
 // 0.0, 4.0 and 10.0
 // Center is biased by 0.5 to include more pixel centers in the
 // thin miters
-constexpr SkScalar x_off_0 = RenderCenterX + 0.5;
-constexpr SkScalar x_off_l1 = x_off_0 - Miter4DiamondOffsetX;
-constexpr SkScalar x_off_l2 = x_off_l1 - Miter10DiamondOffsetX;
-constexpr SkScalar x_off_l3 = x_off_l2 - Miter10DiamondOffsetX;
-constexpr SkScalar x_off_r1 = x_off_0 + Miter4DiamondOffsetX;
-constexpr SkScalar x_off_r2 = x_off_r1 + MiterExtremeDiamondOffsetX;
-constexpr SkScalar x_off_r3 = x_off_r2 + MiterExtremeDiamondOffsetX;
-constexpr SkPoint VerticalMiterDiamondPoints[] = {
+constexpr SkScalar kX_off_0 = kRenderCenterX + 0.5;
```

Comment:
The k and snake casing looks kind of weird, in another PR if you want you can enforce CamelCase, up to you though.
```
  - key: readability-identifier-naming.GlobalConstantCase
    value: CamelCase
```

Reply:
Agreed. I was a bit conflicted on this one and I suspect the right answer is to force us to rename these to something a bit clearer.

Reply:
Done. Or will be done as soon as the presubmits finish and the commit pushes :)